frontend/src/index.tsx

```diff
@@ -77,8 +79,15 @@ function afterLogin(email: string) {
         cursor.subscribeToCloudProcessingServerPongs();
 
         // Check if profile.json file is empty.
-        if([...cursor.myProfileJson.values.values()].every(value => value === ''))
-          showToast("We have noticed you have not created a profile yet. Please go to your profile at top right and create it!.", "info");
+        if([...cursor.myProfileJson.values.values()].every(value => value === '')) {
```

Comment:
Like we talked about. Otherwise LGTM

Reply:
I just added some tests to the if to avoid errors and a more detailed explanation of what it is doing.